src/IP.php

```diff
@@ -6,252 +6,266 @@
  * For the full copyright and license information, please view the LICENSE file.
  *
  * @author Rémi Lanvin <remi@cloudconnected.fr>
- * @link https://github.com/rlanvin/php-ip 
+ *
+ * @see https://github.com/rlanvin/php-ip
  */
 
-if ( ! function_exists('gmp_shiftl') ) {
-	/**
-	 * Shift left (<<)
-	 * @link http://www.php.net/manual/en/ref.gmp.php#99788
-	 */
-	function gmp_shiftl($x, $n)
-	{
-		return gmp_mul($x, gmp_pow('2', $n));
-	}
+namespace phpIP;
```

Comment:
I believe it's somewhat standard that the namespace is capitalized no? So I'd go for `PhpIP`.

Reply:
I don't think it's a PSR convention. My reasoning is PHP's branding is lowercase, but I can change the PR.